### ISSUE TITLE
this.addButtons error

### ISSUE THREAD

**Issue by davidcralph** [Mundayne/reaction-core#6]
when trying to use the example, i got this error:

```
    this.addButtons(...buttons)
         ^

TypeError: undefined is not a function
    at new Menu (C:\Users\no\Documents\Code\Node\Discord\reaction-core-master\src\menu.js:6:10)
    at Object.<anonymous> (C:\Users\no\Documents\Code\Node\Discord\reaction-core-master\example.js:22:20)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3```

(i haven't modified any code)

**Comment by Mundayne** [comment]
Remove the spread operator (ellipses (...)). Example is broken since v2.1.0 because of these.

**Comment by Mundayne** [comment]
Remove the spread operator (ellipses (...)). Example is broken since v2.1.0 because of these.

**Comment by davidcralph** [comment]
works now, though it only reacts with 1 on the example

**Comment by Mundayne** [comment]
Hey sorry for the really (really (*really*)) late response... Is this error persisting and/or are you even using this anymore?
Cheers.

**Comment by davidcralph** [comment]
Hi there, I've not been using it but am still interested to use this. Last time I checked, the issue is still persisting.

**Comment by Mundayne** [comment]
Mega necro but should be fixed as of 2.3.0

**Comment by davidcralph** [comment]
That's one massive necro. I'll check to see if it works soon, thanks.
